### ISSUE TITLE
failed to collect info from nodes

### ISSUE THREAD

**Issue by cforce** [davidB/kubectl-view-allocations#51]
What is the issue? What rights is missing?
I can describe nodes without any issue but kubectl-view-allocations has issues. 

```
./kubectl-view-allocations  -g namespace
[2020-07-17T06:38:33Z ERROR kubectl_view_allocations] failed
    cli: CliOpts { namespace: None, show_zero: false, resource_name: [], group_by: [resource, namespace] }
    error: failed to collect info from nodes

    Caused by:
        0: Failed to list nodes via k8s api
        1: Error deserializing response
        2: invalid type: null, expected a sequence at line 1 column 27551
```

**Comment by davidB** [comment]
Can you list nodes `kubectl get nodes` ?
I'll release a new version with an updated version of the k8s api lib used by the app.

**Comment by cforce** [comment]
yes, i can

**Comment by davidB** [comment]
I released v0.9.1 with an update of library used to connect to k8s. Maybe it will works or provide more information about the issue.
